Libraries/plist/CMakeLists.txt

```diff
@@ -69,6 +69,10 @@ add_executable(plutil Tools/plutil.cpp)
 target_link_libraries(plutil plist util)
 install(TARGETS plutil DESTINATION usr/bin)
 
+add_executable(PlistBuddy Tools/PlistBuddy.cpp)
+target_link_libraries(PlistBuddy plist util edit)
```

Comment:
I don't think libedit is available on Linux, and I'd assume you'd want to talk to lawyers before touching readline, so maybe add a submodule for https://github.com/antirez/linenoise or https://github.com/troglobit/editline?